app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  before_save{email.downcase!}
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-]+(\.[a-z\d\-]+)*\.[a-z]+\z/i
+  validates :name,  presence: true, length: {maximum: 50}
```

Comment:
1. Sau dấu phẩy thì 1 space thôi em nhé

2. `50`, `255`, ..
em khai báo vào file Settings nhé
Để làm được thì em tìm hiểu `gem config in rails `nhé